lib/i18n.py

```diff
@@ -7,26 +7,22 @@
 import os
 import sys
 from os.path import dirname, realpath
+from typing import Callable, Tuple
 
 from .utils import cache
 
-_ = translation = None
-locale_dir = None
-
 # Use N_ to mark a string for translation but _not_ immediately translate it.
 # reference: https://docs.python.org/3/library/gettext.html#deferred-translations
 # Makefile configures pybabel to treat N_() the same as _()
 
 
-def N_(message): return message
-
+def N_(message: str) -> str: return message
```

Comment:
Let's go ahead and put the return on its own line.  I know this one's my doing in the first place, but now it just looks wrong to me :)